resources/application-connector/values.yaml

```diff
@@ -42,31 +42,31 @@ global:
       version: "PR-13670"
     application_connectivity_certs_setup_job:
       name: "application-connectivity-certs-setup-job"
-      version: "PR-13170"
+      version: "708f6a87"
```

Comment:
PR `PR-13170` is merged as a commit [cc89c54](https://github.com/kyma-project/kyma/commit/cc89c542114a448a9a1557e1ea98633c4639f25f). Therefore, the bumped image should be `cc89c542`